build-push-featurebranch.sh

```diff
@@ -1,15 +1,22 @@
 #!/bin/bash
 set -e
 
-SERVER_NAME=mizu
 GCP_PROJECT=up9-docker-hub
 REPOSITORY=gcr.io/$GCP_PROJECT
-GIT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2 | tr '[:upper:]' '[:lower:]')
-SEM_VER=${SEM_VER=0.0.0}
+SERVER_NAME=mizu
+
+if [ "$CI" != 'true' ]
```

Comment:
can separate this file to multiple files, and use it without additional flags (like $CI).
For instance a script which builds the image, a script which push the image, a script which builds and push.
in make docker run only the build img script...